crates/pop-cli/tests/parachain.rs

```diff
@@ -172,37 +193,20 @@ name = "collator-01"
 			"--suri",
 			"//Alice",
 			"--skip-confirm",
-		])
-		.assert()
-		.success();
+		],
+	);
+	assert!(command.spawn()?.wait()?.success());
 
-	assert!(cmd.try_wait().unwrap().is_none(), "the process should still be running");
+	assert!(up.try_wait()?.is_none(), "the process should still be running");
 	// Stop the process
-	Cmd::new("kill").args(["-s", "TERM", &cmd.id().to_string()]).spawn()?;
+	Command::new("kill").args(["-s", "SIGINT", &up.id().to_string()]).spawn()?;
 
 	Ok(())
 }
 
-fn generate_all_the_templates(temp_dir: &Path) -> Result<()> {
-	for template in Parachain::VARIANTS {
-		let parachain_name = format!("test_parachain_{}", template);
-		let provider = template.template_type()?.to_lowercase();
-		// pop new parachain test_parachain --verify
-		Command::cargo_bin("pop")
-			.unwrap()
-			.current_dir(&temp_dir)
-			.args(&[
-				"new",
-				"parachain",
-				&parachain_name,
-				&provider,
-				"--template",
-				&template.to_string(),
-				"--verify",
-			])
-			.assert()
-			.success();
-		assert!(temp_dir.join(parachain_name).exists());
-	}
-	Ok(())
+fn pop(dir: &Path, args: impl IntoIterator<Item = impl AsRef<OsStr>>) -> Command {
+	let mut command = Command::new(cargo_bin("pop"));
```

Comment:
Use normal command rather than the one provided by assert_cmd in order to get maximum ouput from the test, useful in debugging CI issues. Integration tests can be flaky.